composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
Ok, both of your message make a lot of sense. I'm now for adding this as a required dependency :D.

Envoyé de mon iPhone

> Le 6 avr. 2015 à 19:04, Daniel Gimenes <notifications@github.com> a écrit :
> 
> In composer.json:
> 
> >      },
> >      "require-dev": {
> >          "phpunit/PHPUnit": "3.7.*",
> >          "satooshi/php-coveralls": ">=0.6.0",
> > +        "psr/http-message": "1.0.*@dev",
> It should be explicit. Consider that psr/http-message would have multiple versions. How can I know which version of psr-7 you want me to implement?
> 
> —
> Reply to this email directly or view it on GitHub.
>